MovieApp/MovieApp/FeedDownloader.m

```diff
@@ -0,0 +1,128 @@
+#import "FeedDownloader.h"
+
+@interface FeedDownloader ()
+{
+    NSMutableArray *_items;
+    NSMutableData *_responseData;
+    
+    NSXMLParser *parser;
+    NSMutableArray *feeds;
+    NewFeedsItem *newItem;
+    NSMutableString *title;
+    NSMutableString *link;
+    NSString *xmlElement;
+    NSMutableString *descripition;
+    
+}
+@property (nonatomic, assign) id<FeedDataReceiver> delegate;
+@end
+
+@implementation FeedDownloader
+
+//making connection between our downloader and caller
+- (void)downloadNewsFromFeed:(NSURL *)feedUrl andReturnTo:(id)dataHandler
+{
+    _delegate=dataHandler;
+    _feedURL=feedUrl;
+    
+    NSURLRequest *request = [NSURLRequest requestWithURL:feedUrl];
+    NSURLConnection *conn = [[NSURLConnection alloc] initWithRequest:request delegate:self];
```

Comment:
Compiler says that `initWithRequest:delegate:` method is depricated since 9.0. Please, change this call as suggested.